backend/app/environments/schema.py

```diff
@@ -18,27 +17,43 @@ class GetEnvironment(Environment):
 
 
 class _AWSS3Config(ORMModel):
-    account_id: int
-    name: str
+    identifier: str
+    bucket_name: str
     arn: str
-    kms: str
+    kms_key: str
+    is_default: bool
 
 
-class Config(RootModel[Dict[str, _AWSS3Config]]):
-    pass
+class _AWSGlueConfig(ORMModel):
+    identifier: str
+    database_name: str
+    bucket_name: str
+    s3_path: str
 
 
 class EnvPlatformServiceConfig(ORMModel):
-    config: Config
+    config: list[_AWSS3Config | _AWSGlueConfig]
```

Comment:
will we save this in the database as a single entry per environment and platform? Or will we also split per service?
Here you chose to do it as a single entry, why did you choose that and not the other option?

Reply:
It is env,  platform, and service granularity. 

class EnvPlatformServiceConfig(Base, BaseORM):
    __tablename__ = "env_platform_service_configs"

    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
    environment_id: Mapped[UUID] = mapped_column(ForeignKey("environments.id"))
    platform_id: Mapped[UUID] = mapped_column(ForeignKey("platforms.id"))
    service_id: Mapped[UUID] = mapped_column(ForeignKey("platform_services.id"))
    config = Column(String)

Reply:
It is a list because it can be multiple identifiers fo e.g. s3